pkg/reconcile/pipeline/context/service_test.go

```diff
@@ -43,8 +47,55 @@ var _ = Describe("Service", func() {
 		Expect(res).To(BeNil())
 	})
 
+	It("should return owned resources", func() {
```

Comment:
There is a scenario where the `OwnedResources` could return an error. A unit test would be nice.
Use this function to return error: https://pkg.go.dev/k8s.io/client-go/testing#Fake.PrependReactor
There is an example in PR #971

Reply:
Added.